chall01/llahti.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+import sys
+
+morse = [".-",
+"-...",
+"-.-.",
+"-..",
+".",
+"..-.",
+"--.",
+"....",
+"..", 
+".---",
+"-.-",
+".-..",
+"--",
+"-.",
+"---",
+".--.",
+"--.-",
+".-.",
+"...",
+"-",
+"..-",
+"...-",
+".--",
+"-..-",
+"-.--",
+"--.."]
+
+if len(sys.argv) != 2 or sys.argv[1] == "":
+    print("usage: ./xlogin.py <a-zA-Z string>")
```

Comment:
you were supposed to write your login.py not `llahti.py`

you could have retrieved this information with `sys.argv[0]`